cloudformation.yml

```
@@ -0,0 +1,18 @@
+      InstanceType: "t2.micro"
+      KeyName: "MyKeyPair" # Potential Issue: Hardcoded sensitive value
+      ImageId: "ami-0abcdef1234567890" # Potential Issue: Outdated AMI
+      SecurityGroups: 
+        - Ref: "InsecureSecurityGroup"
+
+  InsecureSecurityGroup:
+    Type: "AWS::EC2::SecurityGroup"
+    Properties:
+      GroupDescription: "Allow all inbound traffic"
+      SecurityGroupIngress:
+        - IpProtocol: "-1" # Potential Issue: Open to all protocols
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)